examples/cisco/demo_trm.pp

```diff
@@ -49,6 +49,12 @@
         route_target_export_mvpn      => $rt_export_mvpn,
         route_target_both_auto_mvpn   => $rt_both_mvpn,
       }
+
+      cisco_ip_multicast { 'default':
+        ensure =>                 present,
+        overlay_distributed_dr => 'true',
+        overlay_spt_only =>       'true',
```

Comment:
Make sure to enable `nv overlay evpn` in the node_utils for `overlay_spt_only`

Reply:
Done.